documentation/docs/extensibility.md

```diff
@@ -1,86 +1,199 @@
+return this
+```
+
+This example can be adapted for other linters of your choice.
+
+## 2) Modified ready-made pipeline
+
+This option describes how you can copy and paste one of the centrally maintained pipelines to make changes not possible otherwise.
+For example, you can't change the order of stages, change which stages run in parallel or add new stages to a centrally maintained pipeline.
+This might be done for an individual project (in the `Jenkinsfile`), or in a separate git repository so it can be used for multiple projects.
+
+### Single project
+
+The default `Jenkinsfile` of centrally maintained pipelines does nothing except for loading the pipeline and running it.
+This is convenient, but limits which aspects of the pipeline are modifiable.
+
+If you have one project using the pipeline, the easiest way to do this modification is to copy the pipeline into your `Jenkinsfile`.
+
+The basic structure of your `Jenkinsfile` should be like this:
+
+```groovy
+@Library(/* Shared library definition, see ¹ */) _
+
+call script: this
+
+void call(parameters) {
+  // Your pipeline code based on our ready-made pipelines
+}
+```
+
+The actual pipeline code (the `call` method in the listing above) can be found here:
+
+* [General purpose pipeline](https://github.com/SAP/jenkins-library/blob/master/vars/piperPipeline.groovy)
+    * ¹ For this pipeline, you need to load this library: `'piper-lib-os@vINSERT_VERSION_HERE'`
+* [SAP Cloud SDK Pipeline](https://github.com/SAP/cloud-s4-sdk-pipeline-lib/blob/master/vars/cloudSdkPipeline.groovy)
+    * ¹ For this pipeline, you need to load this library: `'s4sdk-pipeline-library@vINSERT_VERSION_HERE'`
+
+For the version identifier, please see the section _How to stay up-to-date_ in this document.
+
+### Multiple projects
+
+If you have multiple projects that share a similar architecture, it might be desirable to share one modified pipeline amongst them.
+Similar to what you can do in an individual `Jenkinsfile`, you can copy the pipeline to your own shared library and modify it.
+
+To do this, create a new git repository in your preferred git hosting service.
+It must be compliant to [how Jenkins shared libraries are built](https://jenkins.io/doc/book/pipeline/shared-libraries/).
+In a nutshell, this means you need a `vars` directory inside which you can place a copy of your preferred pipeline.
+
+A minimal example of such a library could have this directory structure:
+
+```
+./vars/myCustomPipeline.groovy
+./README.md
+```
+
+Where `myCustomPipeline.groovy` contains the modified pipeline code of the [general purpose pipeline](https://github.com/SAP/jenkins-library/blob/master/vars/piperPipeline.groovy) or [SAP Cloud SDK Pipeline](https://github.com/SAP/cloud-s4-sdk-pipeline-lib/blob/master/vars/cloudSdkPipeline.groovy).
+
+!!! note
+    Your custom pipeline _must_ be named differently from the other pipelines provided by project "Piper", because Jenkins requires names across multiple libraries to be unique.
```

Comment:
```suggestion
    The name of your custom pipeline _must_ differ from the other pipelines provided by project "Piper" because Jenkins requires names across multiple libraries to be unique.
```